src/main/java/Homework2/Lorry.java

```diff
@@ -0,0 +1,24 @@
+package Homework2;
+
+public class Lorry extends Vehicle implements CanRunnig{
+
+
+    private String form = "be-eg";
+    private Integer countOfWheels = 16;
+    String speed = "normal";
```

Comment:
fields from and countOfWheels are already defined in superclass Vehicle
you can instantiate them using constructor